x-pack/test/fleet_api_integration/apis/epm/install_bundled.ts

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import fs from 'fs/promises';
+import path from 'path';
+
+import { FtrProviderContext } from '../../../api_integration/ftr_provider_context';
+import { skipIfNoDockerRegistry } from '../../helpers';
+import { setupFleetAndAgents } from '../agents/services';
+
+export default function (providerContext: FtrProviderContext) {
+  const { getService } = providerContext;
+  const supertest = getService('supertest');
+
+  const BUNDLED_PACKAGE_FIXTURES_DIR = path.join(
+    path.dirname(__filename),
+    '../fixtures/bundled_packages'
+  );
+  const BUNDLED_PACKAGES_DIR = path.join(
+    path.dirname(__filename),
+    '../../../../plugins/fleet/target/bundled_packages'
+  );
+
+  const bundlePackage = async (name: string) => {
+    try {
+      await fs.access(BUNDLED_PACKAGES_DIR);
+    } catch (error) {
+      await fs.mkdir(BUNDLED_PACKAGES_DIR);
+    }
+
+    await fs.copyFile(
+      path.join(BUNDLED_PACKAGE_FIXTURES_DIR, `${name}.zip`),
+      path.join(BUNDLED_PACKAGES_DIR, `${name}.zip`)
+    );
+  };
+
+  const removeBundledPackages = async () => {
+    try {
+      const files = await fs.readdir(BUNDLED_PACKAGES_DIR);
+
+      for (const file of files) {
+        await fs.unlink(path.join(BUNDLED_PACKAGES_DIR, file));
+      }
+    } catch (error) {
+      // ignore
```

Comment:
nit: maybe log the error if the unlink fail for any reason it will probably impact other tests and at least we can see that